src/components/Debate/DebateView.tsx

```diff
@@ -52,6 +63,24 @@ const DebateView = ({ claim }) => {
                         title={claim?.title}
                         personalities={claim?.personalities}
                     />
+                    {userRole === Roles.Regular ? null : (
```

Comment:
The user needs to have a permission equals to admin to have the permission to open the edit debate mode